.gitattributes

```diff
@@ -1,4 +1 @@
-**/*.js.snap text eol=lf
-**/__testfixtures__/** text eol=lf
-**/__tests__/fixtures/** text eol=lf
-**/*.md text eol=lf
\ No newline at end of file
+* text=auto eol=lf
```

Comment:
@pieh what do you think of this? I kinda wanted to do this from the beginning, but I can't remember if there was a reason we were so selective?

Reply:
This is the main (and I think only) reason - https://help.github.com/articles/dealing-with-line-endings/#refreshing-a-repository-after-changing-line-endings, but we will need to do it eventually, so might as well rip band-aid off now

Reply:
That sucks so much, ha. But yeah, this is clearer and more maintainable going forward I suppose.